pkg/controller.v1/tensorflow/controller.go

```diff
@@ -446,8 +446,8 @@ func (tc *TFController) reconcileTFJobs(tfjob *tfv1.TFJob) error {
 			now := metav1.Now()
 			tfjob.Status.CompletionTime = &now
 		}
-		err := updateTFJobConditions(tfjob, common.JobFailed, tfJobFailedReason, failureMessage)
-		if err != nil {
+		if err := updateTFJobConditions(
+		tfjob, common.JobFailed, tfJobFailedReason, failureMessage); err != nil {
```

Comment:
I think it does not gofmted. :think:

Reply:
```
if err := updateTFJobConditions(tfjob, common.JobFailed, tfJobFailedReason, failureMessage); err != nil {...}

if err := updateTFJobConditions(
tfjob, common.JobFailed, tfJobFailedReason, failureMessage); err != nil {...}
```

both pass my gofmt, which one is better?

Reply:
```go
	if err := updateTFJobConditions(
		tfjob, common.JobFailed, tfJobFailedReason, failureMessage); err != nil {...}
```

LGTM

The line is too long, we can have a new line. But if we have the new line, we need a tab to keep indent.

Reply:
Done